services/convert/convert.go

```diff
@@ -283,6 +284,8 @@ func ToDeployKey(apiLink string, key *asymkey_model.DeployKey) *api.DeployKey {
 
 // ToOrganization convert user_model.User to api.Organization
 func ToOrganization(ctx context.Context, org *organization.Organization) *api.Organization {
+	numPackages, _ := packages_model.CountLatestVersions(ctx, &packages_model.PackageSearchOptions{OwnerID: org.ID, IsInternal: util.OptionalBoolFalse})
```

Comment:
This change looks really strange and not at all how the code should be structured.
This method **should not** make database queries.
It's a much better idea to store the information inside `organization.Organization`, **if at all**:
After all, there are already API routes for getting this information (-> `X-Total-Count`).
Furthermore, if we do that, I can already see the requests to also add it for issues, labels, milestones, PRs, releases, tags, projects, …
So in summary, I'm against modifying the API.

Reply:
To the API response in general: The Number is shown on the Profile, so it should also be returned by the API, so 3rd Party Apps can add this Feature easily. As we don't show Issues or Milestones on the profile, there is no need to return a count.

To the location: There are also other places in the convert API ehere databse querrys are done, but I can move it to the User7org Modell and add a `LoadCount` function.